src/Azos/Standards/Distance.cs

```diff
@@ -361,8 +361,8 @@ void IJsonWritable.WriteAsJson(TextWriter wri, int nestingLevel, JsonWritingOpti
     public static Distance operator +(Distance a, Distance b) => new Distance(a.Unit, a.ValueInMicrons + b.ValueInMicrons);
     public static Distance operator -(Distance a, Distance b) => new Distance(a.Unit, a.ValueInMicrons - b.ValueInMicrons);
 
-    public static Area     operator *(Distance a, Distance b) => new Area(a.Unit, a.ValueInMicrons * b.ValueInMicrons);
-    public static decimal  operator /(Distance a, Distance b) => (decimal)a.ValueInMicrons / (decimal)b.ValueInMicrons;
+    public static Distance operator *(Distance a, Distance b) => new Distance(a.Unit, a.ValueInMicrons * b.ValueInMicrons);
```

Comment:
I think, this is wrong. Multiplying two distances should yield Area.
Multiplying a distance by a number should yield a distance, but distance * distance = area